templates/shared/_cloud-contact-us-form.html

```diff
@@ -1,43 +1,391 @@
+                <span class="p-checkbox__label" id="i-dont-know">I don't know</span>
+              </label>
+            </div>
+          </div>
+        </fieldset>
+        <fieldset class="u-no-margin--bottom" id="kind-of-device" aria-required="true">
+          <legend class="u-off-screen">*What kind of device are you using?</legend>
+          <h2 class="p-heading--3">*What kind of device are you using?</h2>
+          <div class="row u-no-padding">
+            <div class="col-4 u-sv3">
+              <label class="p-checkbox">
+                <input required
+                       class="p-checkbox__input"
+                       type="checkbox"
+                       aria-labelledby="desktop/workstation"
+                       value="desktop/workstation"
+                       onclick="validateCheckbox(event, 'kind-of-device')" />
+                <span class="p-checkbox__label" id="desktop-workstation">Desktop/workstation</span>
+              </label>
+              <label class="p-checkbox">
+                <input type="checkbox"
+                       aria-labelledby="physical server"
+                       class="p-checkbox__input"
+                       value="physical server"
+                       onclick="validateCheckbox(event, 'kind-of-device')" />
+                <span class="p-checkbox__label" id="physical-server">Physical server</span>
+              </label>
+              <label class="p-checkbox">
+                <input type="checkbox"
+                       aria-labelledby="public-cloud"
+                       class="p-checkbox__input"
+                       value="public cloud"
+                       onclick="validateCheckbox(event, 'kind-of-device')" />
+                <span class="p-checkbox__label" id="public-cloud">Public cloud</span>
+              </label>
+              <label class="p-checkbox">
+                <input type="checkbox"
+                       aria-labelledby="virtual-machine"
+                       class="p-checkbox__input"
+                       value="virtual machine"
+                       onclick="validateCheckbox(event, 'kind-of-device')" />
+                <span class="p-checkbox__label" id="virtual-machine">Virtual machine</span>
+              </label>
+              <label class="p-checkbox">
+                <input type="checkbox"
+                       aria-labelledby="iot-edge-device"
+                       class="p-checkbox__input"
+                       value="iot/edge device"
+                       onclick="validateCheckbox(event, 'kind-of-device')" />
+                <span class="p-checkbox__label" id="iot-edge-device">IoT/Edge device</span>
+              </label>
+            </div>
+          </div>
+        </fieldset>
+        <fieldset class="u-no-margin--bottom" id="how-many-machines">
+          <legend class="u-off-screen">*How many devices?</legend>
+
+          <h2 class="p-heading--3">
+            *How many
+            {% if howManyVM %}
```

Comment:
This condition is wrong, the value assigned is a string `"true"`, so `"false"` would be truthy. Either replace this line to `howManyVM === "true"` or the value you pass in the include should be a boolean:
```
    ....
    formid="3473",
    lpId="",
    returnURL="/aws/thank-you",
    howManyVM=True %}
```

Notice the boolean howManyVM `True`